.circleci/config.yml

```diff
@@ -28,11 +28,15 @@ jobs: # a collection of steps
           path: frontend/coverage
           destination: frontend-coverage
       - run:
-          name: install backend
+          name: build backend
```

Comment:
We should clean the dist folder before building. See [here](https://webpack.js.org/guides/output-management/#cleaning-up-the-dist-folder).

This is needed especially in the context of this pull request; otherwise, the old/cached source map files will be packaged into the .vsix file.

Reply:
Not needed for Circle CI: It turns out that the dist folder is not cached.